app/templates/static/layout.static.coco.pug

```diff
@@ -3,7 +3,7 @@ block begin
 html(lang='en')
   head
     meta(charset='utf-8')
-    meta(name='viewport', content='width=1024', data-vue-meta="true", data-vmid="viewport")
+    meta(name='viewport', content="width=device-width,initial-scale=1.0", data-vue-meta="true", data-vmid="viewport")
```

Comment:
Is this tested–what impact does it have?

Reply:
I tested on various screen sizes and everything seems ok. This is a standard HTML5 convention for viewport so that it takes the width of the device user is on. Also, it is one of the requirement to get Toggle Device Toolbar to work, though I haven't been able to work all of its features yet after adding this.